src/core/plugins/oas31/oas3-extensions/fn.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @prettier
+ */
+import { immutableToJS } from "core/utils"
+
+export const makeGetIsFileFormat = (fn) => {
+  const getIsFileFormat = (schema) => {
+    if (!schema) return null
+
+    const type = fn.jsonSchema202012.getType(immutableToJS(schema))
+    const isFileContentMediaType = fn.getIsFileContentType(
+      schema.get("contentMediaType")
+    )
+    const isBase64Encoding = schema.get("contentEncoding") === "base64"
+
+    return (
+      (type === "string" || type === "any") &&
```

Comment:
Comment author @glowcloud:

If the `type` is not specified, we will get type `any` because there is no check for `contentEncoding` or `contentMediaType` when. I see that other types also don't have a check for all of the keywords that apply to them - not sure if that is intended or not, so I opted for checking for `any` instead of adding them there.